apertium_apy/apy.py

```diff
@@ -215,7 +228,7 @@ def parse_args(cli_args=sys.argv[1:]):
                         help='keeps unknown words in memory until a limit is reached; use with --missing-freqs (default = 1000)',
                         type=int, default=1000)
     parser.add_argument('-T', '--stat-period-max-age',
-                        help='How many seconds back to keep track request timing stats (default = 3600)', type=int, default=3600)
+                        help='How manAy seconds back to keep track request timing stats (default = 3600)', type=int, default=3600)
```

Comment:
?

Reply:
No idea how this creeped in. Sorry for this